MapboxDirections/MBRouteLeg.swift

```diff
@@ -98,6 +130,9 @@ open class RouteLeg: NSObject, NSSecureCoding {
      */
     open let steps: [RouteStep]
     
+    
+    open let annotation: [AnnotationType:[Any]]?
```

Comment:
With a split between "node attributes" and "segment attributes", we'll eventually be able to distribute the attribute values among each step (while still retaining the full, leg-level attribute structure as an option).